### PR TITLE
Add job info to sentry tags for webhook jobs

### PR DESCRIPTION
**Pull request by danlamanna**

### CODE REVIEW

File: analytics/analytics/build_timing_processor.py [hunk 1]
```diff
@@ -88,6 +90,7 @@ def get_timings_json(job: ProjectJob) -> list[dict]:
 def upload_build_timings(job_input_data_json: str):
     # Read input data and extract params
     job_input_data = json.loads(job_input_data_json)
+    setup_gitlab_job_sentry_tags(job_input_data)
```
Comment (jjnesbitt) [inline analytics/analytics/build_timing_processor.py:93]: Should we not just put the invocation to this function in the webhook endpoint itself?

Comment (danlamanna) [reply analytics/analytics/build_timing_processor.py:93]: It'll be run on a different machine. I briefly tried looking into how to reduce the duplication, I'd probably use a custom celery base task, but it didn't really look easier 😕.

Comment (jjnesbitt) [reply analytics/analytics/build_timing_processor.py:93]: Right, I see.

File: .github/workflows/custom_docker_builds.yml [hunk 2]
```diff
@@ -46,7 +46,7 @@ jobs:
           - docker-image: ./images/cache-indexer
             image-tags: ghcr.io/spack/cache-indexer:0.0.3
           - docker-image: ./analytics
-            image-tags: ghcr.io/spack/django:0.0.3
+            image-tags: ghcr.io/spack/django:0.0.4
```
Comment (jjnesbitt) [inline .github/workflows/custom_docker_builds.yml:49]: This needs to be bumped in https://github.com/spack/spack-infrastructure/blob/main/k8s/production/custom/webhook-handler/deployments.yaml as well right?

Comment (danlamanna) [reply .github/workflows/custom_docker_builds.yml:49]: 😩

**Review by jjnesbitt — APPROVED** [review]

**Pull request merged**